src/ir/subtypes.h

```diff
@@ -18,7 +18,7 @@
 #define wasm_ir_subtypes_h
```

Comment:
github informs me that something changed in this file since last I read it. Looks like there were forced-pushes, so I can't read the commit, and I tried the "show diff" on the force-push, which @brendandahl recommended to me a while ago. But the diffs there is huge and dominated by unrelated changes, merges from main, I assume.

Am I missing a way that github marks the changes in the PR from unrelated changes? Or is there another good way to see incremental updates in a large PR like this with force-pushes?

Reply:
No, sorry, this was a more destructive force push than usual because there were many merge conflicts when rebasing on main. The only way to avoid that would be to use merge commits instead of rebasing, but that comes with its own headaches.

Reply:
To navigate stacked PRs effectively, you want to maintain a linear history from the tip of the stack back down to `main`. As such, my stack management script current [rebases](https://github.com/tlively/git-tl/blob/main/git-tl#L410-L420) when syncing new changes, which keeps the stack organization the same as the underlying commit organization. I could experiment with maintaining the linear stack of PRs but allowing the underlying commits to become arbitrary merge spaghetti, though. Maybe it wouldn't actually affect my workflow.

Reply:
I don't have much trouble with stacked PRs myself, and I never rebase. Though, I rarely open multiple parts of the stack on github at once? (I use branches in my personal repo, which would make later parts PRs on my fork, not upstream. So I just wait to open them.)

Each time I make a change in a PR in the middle of the stack, I need to update the ones "downstream" in the chain. I just do merge commits for those. I have had no issues when doing so.

Reply:
I updated my script to use merges instead of rebases. Let's see what happens!